app/utils/createEntityReducer.js

```diff
@@ -158,6 +160,7 @@ export function deleteEntities(deleteTypes: ?EntityReducerTypes) {
               ? [Number(resultId), resultId.toString()]
               : [resultId])
           ),
+          // eslint-disable-next-line no-sequences
```

Comment:
Maybe rewrite this instead?

Reply:
Also unsure of the syntax here :slightly_smiling_face:, but I think the rewrite should be about right. :upside_down_face: